components/Contact.js

```diff
@@ -1,7 +1,83 @@
-import React from "react";
+import React, { useState } from "react";
 import userData from "@constants/data";
+import emailjs from "@emailjs/browser";
+
+function ContactForm(props) {
```

Comment:
Love the component extraction - nice work! 👌 

One suggestion: wdyta moving this to its own file? eg `ContactForm.js`?

The existing codebase idiom seems to be one-component-per-file, so seems reasonable to stay consistent.

Reply:
Good idea! Done! Noticed a couple places elsewhere in the code where this doesn't occur, so I will add it to the backlog to make this uniform.